mappings/net/minecraft/client/resource/language/LanguageDefinition.mapping

```diff
@@ -1,12 +1,12 @@
 CLASS net/minecraft/class_1077 net/minecraft/client/resource/language/LanguageDefinition
 	FIELD field_5326 code Ljava/lang/String;
-	FIELD field_5327 region Ljava/lang/String;
+	FIELD field_5327 name Ljava/lang/String;
 	FIELD field_5328 rightToLeft Z
-	FIELD field_5329 name Ljava/lang/String;
+	FIELD field_5329 region Ljava/lang/String;
```

Comment:
I don't think this applies even a few snapshots before the current one. Mind check in 1.16.5?

Reply:
`LanguageDefinition` implements `Language` which is a non-obfuscated interface, it has three getters including the name and region getters.

Looking at the implementation it was previously mismapped as `getName()` returned `region` before and `getRegion` returned `name`, I believe that this backporting is correct and act as a fix.